pkg/dao/repository_configs.go

```diff
@@ -387,6 +387,66 @@ func (r repositoryConfigDaoImpl) Delete(orgID string, uuid string) error {
 	return nil
 }
 
+func (r repositoryConfigDaoImpl) BulkDelete(orgID string, uuids []string) []error {
+	var responses []api.RepositoryResponse
+	var errs []error
+
+	_ = r.db.Transaction(func(tx *gorm.DB) error {
+		var err error
+		responses, errs = r.bulkDelete(tx, orgID, uuids)
+		if len(errs) > 0 {
+			err = errors.New("rollback bulk delete")
+		}
+		return err
+	})
+
+	mappedValues := make([]repositories.Repositories, len(uuids))
+	for i := 0; i < len(responses); i++ {
+		mappedValues = append(mappedValues, notifications.MapRepositoryResponse(responses[i]))
+	}
+	notifications.SendNotification(orgID, notifications.RepositoryDeleted, mappedValues)
```

Comment:
```suggestion
	mappedValues := make([]repositories.Repositories, len(uuids))
	for i := 0; i < len(responses); i++ {
		mappedValues[i] = notifications.MapRepositoryResponse(responses[i])
	}
	if len(mappedValues) > 0 {
	        notifications.SendNotification(orgID, notifications.RepositoryDeleted, mappedValues)
	}
```
A couple of things here. And you don't have to change it exactly this way, but I think showing the difference is the best way to explain it.

1. I changed the list append syntax to the indexing because the `make` function allocates a list of empty values. By appending, we'd be appending to that empty list, instead of replacing, causing blank notifications to go out.
2. We want to check that mappedValues has values to send out, otherwise we'd be sending out an empty notification even when there is an error.